ERP/resources/views/components/navbar.blade.php

```diff
@@ -15,6 +15,9 @@
                 </li>
                 @endif
                 @if(Auth::user() != null)
+                <li class="nav-item">
+                    <a class="nav-link active" href="/product-manager">Product Manager's Page</a>
```

Comment:
Instead of naming it "Product Manager's Page", should we just name it "Product Manager" as it will take less space on the navbar?

Reply:
its a place holder for now, but i agree, the name is a little long, we could shorten it even more